src/ansi.rs

```diff
@@ -677,6 +677,9 @@ pub trait Handler {
     ///
     /// The output is of form `CSI > 4 ; mode m`.
     fn report_modify_other_keys(&mut self) {}
+
+    // Set SCP control
+    fn set_scp(&mut self, _ps1: u16, _ps2: u16) {}
```

Comment:
`ps1` and `ps2` are not appropriate variable names. Instead the actual purpose of the variables should be used. This applies to the whole PR.